src/components/ha-push-notifications-toggle.js

```diff
@@ -93,7 +106,25 @@ class HaPushNotificationsToggle extends EventsMixin(PolymerElement) {
         return;
       }
 
-      sub = await reg.pushManager.subscribe({ userVisibleOnly: true });
+      let applicationServerKey = null;
+      try {
+        const res = await this.hass.callApi(
+          "GET",
+          "notify.html5/applicationServerKey"
+        );
+        applicationServerKey = res.message;
+      } catch (ex) {
+        applicationServerKey = null;
+      }
+
+      if (applicationServerKey) {
+        sub = await reg.pushManager.subscribe({
+          userVisibleOnly: true,
+          applicationServerKey: urlBase64ToUint8Array(applicationServerKey),
```

Comment:
Is applicationServerKey baes64 encoded?

Reply:
Apparently some kind of urlsafe base64 encoding, look at the function at the top.